js/debug/job_recommendation.js

```diff
@@ -0,0 +1,55 @@
+(function ($) {
+    var click = function(jobId, recommender){
+      if(!JOBCENTRE.capabilities.localStorage)
+          return;
+
+      var click = {'jobId': parseInt(jobId), 'recommender': recommender, 'timestamp': new Date().toISOString()};
+
+      var clickArray = JSON.parse(window.localStorage.getItem('recommendationInfo'));
+      if(clickArray == null)
+          clickArray = [];
+
+      var clickIndex = clickArray.map(function(e){return e.jobId;}).indexOf(parseInt(jobId));
```

Comment:
We haven't polyfilled `.map`, so will break in older browsers.  Better to use underscore's `_.map()` (http://underscorejs.org/#map) which is loaded on every page.

Reply:
We should always pass in the `radix` to `parseInt`:

```
parseInt(jobId, 10)
```

More info here: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/parseInt

![image](https://user-images.githubusercontent.com/504505/29797807-287d0c98-8c0e-11e7-80c3-eb8f7d6f7071.png)